lib/browser/chrome-extension.js

```diff
@@ -201,6 +198,34 @@ ipcMainInternal.on('CHROME_TABS_SEND_MESSAGE', function (event, tabId, extension
   resultID++
 })
 
+const getLanguage = () => {
+  return app.getLocale().replace(/-.*$/, '').toLowerCase()
```

Comment:
`app.getLocale()` and `navigator.language` aren't always the same thing right?

Reply:
@MarshallOfSound I am not sure. I wanted to avoid passing the language via IPC as it would have to be validated.